kv/db.go

```diff
@@ -163,6 +165,71 @@ func (s *rpcDBServer) executeCmd(args proto.Request, reply proto.Response) error
 	return nil
 }
 
+func (s *rpcDBServer) CreateTable(args *proto.CreateTableRequest, reply *proto.CreateTableResponse) error {
```

Comment:
A bit strange for the `CreateTable` RPC to be handled in the `kv` package. Probably belongs in `structured/db.go` instead. We might want to strip out the existing code from `structured` in order to start with a clean slate.